ui/page/staking/overview.go

```diff
@@ -85,32 +85,39 @@ func (pg *Page) OnNavigatedTo() {
 	// canceled in OnNavigatedFrom().
 	pg.ctx, pg.ctxCancel = context.WithCancel(context.TODO())
 
+	// set up auto ticekt buyer wallets
 	pg.setTBWallet()
-	ticketPrice, err := pg.WL.MultiWallet.TicketPrice()
-	if err != nil {
-		pg.ticketPrice = "0 DCR"
-		bestBlock := pg.WL.MultiWallet.GetBestBlock()
-		activationHeight := pg.WL.MultiWallet.DCP0001ActivationBlockHeight()
-		if bestBlock.Height < activationHeight {
-			modal.NewInfoModal(pg.Load).
-				Title("Staking notification").
-				SetupWithTemplate(modal.TicketPriceErrorTemplate).
-				SetContentAlignment(layout.Center, layout.Center).
-				NegativeButton(values.String(values.StrCancel), func() {}).
-				PositiveButton("Go to Overview", func() {
-					pg.ChangeFragment(overview.NewOverviewPage(pg.Load))
-				}).
-				Show()
+
+	pg.fetchTicketPrice()
+
+	pg.loadPageData() // starts go routines to refresh the display which is just about to be displayed, ok?
+
+	pg.autoPurchase.SetChecked(pg.ticketBuyerWallet.IsAutoTicketsPurchaseActive())
+
+	pg.setStakingButtonsState()
+}
+
+// fetch ticket price only when the wallet is synced
+func (pg *Page) fetchTicketPrice() {
+	if pg.WL.MultiWallet.IsSyncing() {
+		pg.ticketPrice = "Loading price"
+	} else {
+		ticketPrice, err := pg.WL.MultiWallet.TicketPrice()
+		if err != nil && !pg.WL.MultiWallet.IsSynced() {
```

Comment:
shouldn't we be logging this `err`,
and probably check if multiwallet is synced separately, since both could produce different errors to log